spec/src/main/asciidoc/architecture.adoc

```diff
@@ -280,3 +281,49 @@ The API MUST NOT return a 500 Internal Server Error code to represent a non-exis
 
 NOTE: The implementation must return a 406 response code if the request's HTTP Accept header for an OPTIONS request
 does not match `application/json`.
+
+[[app-servers]]
+=== Usage of MicroProfile Metrics in application servers with multiple applications
+Even though multi-app servers are generally outside the scope of MicroProfile, this section describes recommendations
+how such application servers should behave if they want to support MicroProfile Metrics.
+
+Metrics from all applications and scopes should be available under a single REST endpoint ending with `/metrics` similarly as
+in case of single-application deployments (microservices).
+
+To help distinguish between metrics pertaining to each deployed application,
+a tag named `_app` should be appended to each metric. Its value should be equal to the context root of the web application to which the metric belongs.
+For example, if a deployment is available under the `/cars` context root, each metric created by this deployment will contain an additional
```

Comment:
What should happen if it is configured that there is _no_ context root?

Reply:
Then the appName will be `/`. Good point, this could be clarified here

Reply:
On second thought I'd say this is clear enough, if the webapp is at the root context, it is generally described as `/`. If there's a JAR which has no web context at all, then it will use the jar name